deployment/v2/4_createRollup.ts

```diff
@@ -94,6 +100,13 @@ async function main() {
                 throw new Error(`Missing sovereign parameter: ${parameterName}`);
             }
         }
+
+        // check aggchainParams if consensusContract is Aggchain
+        if (consensusContract.includes("Aggchain")) {
```

Comment:
Sometimes there is this `toLowerCase`, sometimes it is not.
I would make an array of aggchains like:
```
const arraySupportedAggchains = [`AggchainECDSA`, `AggchainFEP`];
```
This array will be appended to the `supportedConsensus`. And then you can check uniquelly if it is an aggchain if it is included in this `arraySupportedAggchains`.

In this way, you also avid any issue if the input contains the word `aggchain` but it doe snot exist its consensus